components/Members/MembersTabs.tsx

```diff
@@ -0,0 +1,42 @@
+import { PropsWithoutRef, SetStateAction, useState } from 'react';
+import { Tabs, Tab } from 'react-bootstrap';
+
+import { Member } from '../../models/Members';
+import MembersList from './MembersList';
+
+type TabProps = PropsWithoutRef<{
+  list: Member[];
+}>;
+type TabsProps = PropsWithoutRef<{
+  [proppName: string]: TabProps;
+}>;
+export type MembersTabsProps = PropsWithoutRef<{
+  tabs?: TabsProps;
+  list?: Member[];
+}>;
```

Comment:
```suggestion
type TabsData = Record<string, {
  list: Member[];
}>;

export interface MembersTabsProps {
  tabs?: TabsData;
```
只有组件的参数对象才称为 Props，内层数据结构都与 React 无关（下同）